packages/base-map/__mocks__/SelectVehicles.js

```diff
@@ -22,44 +22,46 @@ class SelectVehicles extends MapLayer {
     vehicles: vehicleData
   };
 
-  constructor(props) {
-    super(props);
-
-    this.onOverlayAdded = this.onOverlayAdded.bind(this);
-    this.onOverlayRemoved = this.onOverlayRemoved.bind(this);
-    this.onViewportChanged = this.onViewportChanged.bind(this);
-  }
-
   // these zoom layers control which markers are shown (e.g. closeZoom is where icons are turned on)
   closeZoom = 15;
 
   midZoom = 13;
 
   farZoom = 10;
 
-  onOverlayAdded(e) {
+  /**
+   * The preferred way to write the `onOverlay...` events is shown below.
+   * If you wish to write `onOverlayAdded(e) {...}` instead,
+   * then you must bind that function in the `constructor` using:
+   *     this.onOverlayAdded = this.onOverlayAdded.bind(this);
+   */
```

Comment:
This should be common knowledge for react developers. This comment could be deleted or better yet replaced with something descriptive of how this handler is activated.